packages/core/src/client.ts

```diff
@@ -2866,7 +2879,7 @@ export class MedplumClient extends EventTarget {
     ) {
       const contentLocation = await tryGetContentLocation(response, obj);
       if (contentLocation) {
-        return this.request('GET', contentLocation, { ...options, body: undefined });
+        return this.request(contentLocation, { ...options, body: undefined });
```

Comment:
Need to set `method` to 'GET' in options?

Reply:
https://github.com/medplum/medplum/pull/4065/commits/ae3e0d9e775a0584b29b1abdfcbd6ec046845c96